src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -352,6 +352,21 @@ def setUp(self):
         self.d_log = DaosLog(self.context)
         self.test_log.daos_log = self.d_log
 
+        # import and set env vars from yaml file
+        default_env = self.params.get("default", "/run/ENV/")
+        if default_env is None:
+            print("default_env was not set in yaml file.\n")
```

Comment:
I think you are going to see this for every non-cart test. Not sure if this print is necessary.
Since there are changes in TestWithoutServers in apricot/test.py, could you kick off all fn vm and fn hw tests?

Reply:
a6140368925d586bfd2a6456d05b39fc45e979cf  

Full run with all fn vm and fn hw tests: [build 23](https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-4445/23/).